lib/chef/policy_builder/expand_node_object.rb

```diff
@@ -54,6 +55,21 @@ def initialize(node_name, ohai_data, json_attribs, override_runlist, events)
         @run_list_expansion = nil
       end
 
+      # This method injects the run_context and provider and resource priority
```

Comment:
This first sentence explains 'what', which can be seen by looking at the code - I would much rather know 'why'.  Why does the run_context need to be injected here?